docs/eventing/event-registry.md

```diff
@@ -69,214 +64,199 @@ spec:
-the next topic: How do we actually populate the registry in the first place?
+Now that we know how to discover events using the registry and how we can leverage that information to subscribe to 
+events of interest, let's move on to the next topic: How do we actually populate the registry in the first place? 
 
 - Manual Registration
 
-  In order to populate the registry, a Cluster Configurator can manually
-  register the EventTypes. This means that the configurator can simply apply
-  EventTypes yaml files, just as with any other Kubernetes resource:
+    In order to populate the registry, a cluster configurator can manually register the EventTypes. 
+    This means that the configurator can simply apply EventTypes yaml files, just as with any other Kubernetes resource:
 
-  `kubectl apply -f <event_type.yaml>`
+    `kubectl apply -f <event_type.yaml>`
 
 - Automatic Registration
 
-  As Manual Registration might be tedious and error-prone, we also support
-  automatic registration of EventTypes. Herein, the creation of the EventTypes
-  is done upon instantiation of an Event Source. We currently support automatic
-  Registration of EventTypes for the following Event Sources:
-
-  - CronJobSource
-  - ApiServerSource
-  - GithubSource
-  - GcpPubSubSource
-  - KafkaSource
-  - AwsSqsSource
-
-  Let's look at an example, in particular, the KafkaSource sample we used to
-  populate the registry in our testing cluster. Below is what the yaml looks
-  like.
-
-  ```yaml
-  apiVersion: sources.eventing.knative.dev/v1alpha1
-  kind: KafkaSource
-  metadata:
-    name: kafka-source-sample
-    namespace: default
-  spec:
-    consumerGroup: knative-group
-    bootstrapServers: my-cluster-kafka-bootstrap.kafka:9092
-    topics: knative-demo,news
-    sink:
-      apiVersion: eventing.knative.dev/v1alpha1
-      kind: Broker
-      name: default
-  ```
-
-  If you are interested in more information regarding configuration options of a
-  KafkaSource, please refer to the
-  [KafKaSource example](https://github.com/knative/eventing-sources/tree/master/contrib/kafka/samples).
-
-  For this discussion, the relevant information from the yaml above are the
-  `sink` and the `topics`. We observe that the `sink` is of kind `Broker`. We
-  currently only support automatic creation of EventTypes for Sources instances
-  that point to Brokers. Regarding `topics`, this is what we use for the
-  EventTypes `source` field, which is equal to the CloudEvent source attribute.
-
-  When you `kubectl apply` this yaml, the KafkaSource `kafka-source-sample` will
-  be instantiated, and two EventTypes will be added to the registry (as there
-  are two topics). You can see that in the registry example output from the
-  previous sections.  
-
-
+    As Manual Registration might be tedious and error-prone, we also support automatic registration of EventTypes. 
+    Herein, the creation of the EventTypes is done upon instantiation of an Event Source. 
```

Comment:
What does "upon instantiation" of an Event Source mean? Is that just when an Event Source is installed (kubectle apply)?

Reply:
yes, that is correct!